taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
```

Comment:
Code Efficiency: Use `get_user_model()` to get the user model. This is important because if you decide to change the user model in the future, you won't have to change it everywhere in your code. Use `get_user_model()` instead of directly importing the `Driver` model.